verifier/testreport.py

```diff
@@ -731,7 +731,8 @@ def characterize_datetime_tests(self, test_list, results):
             if 'skeleton' in  test['input_data']:
                 skeleton_str = 'skeleton: ' + test['input_data']['skeleton']
                 results.setdefault(skeleton_str, []).append(label)
-
+            if 'dateTimeFormatType' in test:
+                results.setdefault('dateTimeFormatType: ' + test['dateTimeFormatType'], [])
```

Comment:
do you need to chain a `.append(label)` to this operation? otherwise, as it is, `.setdefault()` doesn't actually do anything to the map, it just gives a default value operate on when the key doesn't yet exist